### DISCUSSION TITLE
Can't Pair flashed Sonoff Zigbee Bridge(Pro) as a Router in Hubitat!!!

### DISCUSSION THREAD

**Discussion by allansari** [arendst/Tasmota#18265] [Q&A]
Hi

Got my new HE hub C7 before unfortunately before the C-8 was released.

Never mind, after taking decisions with HE,i want to strengthen coverage, so i flashed (Sonoff ZigBee bridge Pro) & trying to make it as a repeater but unfortunately, I couldn't pair it with Hubitat.

i tried to follow all the steps in GutHub and here : [Sonoff Zigbee Bridge Simply as a Router in HE](https://community.hubitat.com/t/sonoff-zigbee-bridge-simply-as-a-router-in-he/90896)

it flashed successfully i assume but can't pair it with Hubitat and link it as a device ??

![image](https://user-images.githubusercontent.com/10684138/227763305-8c793193-43cf-4038-9ad2-d34f8cfd0426.png)

Chip Info:

Chip Family: ESP32
Chip Model: ESP32-D0WDQ5-V3 (revision 3)
Number of Cores: 2
Max CPU Frequency: 240MHz
Has Bluetooth: YES
Has Embedded Flash: NO
Has Factory-Calibrated ADC: YES
MAC Address: 40:22:D8:5F:4E:48
here it shows , flashed successfully

10:57:19.651 FLH: Starting verification of HEX file
10:57:29.898 FLH: Verification of HEX file OK
10:57:33.254 FLH: cc2652_flasher rx=23 tx=19 rst=15 bsl=22
10:57:33.293 FLH: Flashing started (takes 5 minutes during which Tasmota is unresponsive)
11:01:19.704 FLH: Flashing completed: OK
11:01:19.807 FLH: Flash crc32 0x000000 - 0x2FFFF = bytes('1598929A')

Those the files i uploaded

![image](https://user-images.githubusercontent.com/10684138/227763315-e8c2c757-3d83-4076-a38f-30171271164b.png)

are these configurations correct??

![image](https://user-images.githubusercontent.com/10684138/227763323-1ef3cebd-ddec-48e6-b810-f5f9c128d290.png)

![image](https://user-images.githubusercontent.com/10684138/227763610-ed593272-27a7-4d7b-90f7-c70809589243.png)

_Originally posted by @allansari in https://github.com/arendst/Tasmota/discussions/16957#discussioncomment-5430331_

**Comment by Jason2866** [comment]
Moved to discussions, since use as Router is a not supported use case.

**Reply by allansari** [reply]
!!!
